src/tools/common/sss_tools.c

```diff
@@ -339,6 +346,14 @@ errno_t sss_tool_route(int argc, const char **argv,
                 return tool_ctx->init_err;
             }
 
+            ret = tool_cmd_init(tool_ctx, &commands[i]);
+            if (ret != 0) {
```

Comment:
Really a nitpick here and sorry for not catching this one in the first review.
Please, compare against EOK instead of 0 (this change can be done before pushing, actually, no need to re-send the patch just because of this).

I'll run a round of tests manually and run the CI just to be sure it doesn't break anything before officially giving the ACK.